src/context/PluginProvider/PluginProvider.tsx

```diff
@@ -128,11 +128,21 @@ export const PluginProvider = ({ children }: PluginProviderProps): JSX.Element =
     )
 
     setRoutes(pluginRoutes)
-    const knownChainIds = featureFlags.Osmosis
-      ? Object.values(KnownChainIds)
-      : Object.values(KnownChainIds).filter(chainId => chainId !== KnownChainIds.OsmosisMainnet)
 
-    const _supportedChains = Object.values(knownChainIds) as ChainId[]
+    const chainIdFilter: Array<string> = []
```

Comment:
maybe not the best variable name, think `omitttedChainIds` might read better, or something along those lines?